README.md

```diff
@@ -0,0 +1,38 @@
+# Swagger UI Field Formatter
+
+## Introduction
+
+This module provides a field formatter for file fields with allowed file types
+as json or yml, which renders the uploaded json or yml file using the Swagger UI
```

Comment:
Do we want to add "yaml" as well, or is it just understood that "yaml" is included? "yaml" is mentioned in line 30 below.